hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/metadata/SCMMetadataStoreImpl.java

```diff
@@ -67,6 +75,23 @@
  */
 public class SCMMetadataStoreImpl implements SCMMetadataStore {
 
+  public static final Set<DBColumnFamilyDefinition<?, ?>> COLUMN_FAMILIES =
+      new HashSet<>(Arrays.asList(
```

Comment:
Do we need this extra map containing all the tables? They are already stored in the tableMap, so we can iterate it to list out all the tables.

My concern is that if someone later adds a new table, they can easily miss adding it to this map, and then the metric will not automatically appear.

If we drive it off the tableMap, then there is nothing special to do around metrics when adding a new table.

Reply:
I think it might be necessary, when I use the tableMap from the StoreImpl, the metrics class will invoke NullPointerException.